src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocVariableCheck.java

```diff
@@ -196,6 +196,7 @@ public class JavadocVariableCheck
      * Setter to specify the visibility scope where Javadoc comments are checked.
      *
      * @param scope a scope.
+     * @since 6.0
```

Comment:
3.0

But we will catch it in validation

Reply:
Done.